Dockerfile

```diff
@@ -0,0 +1,21 @@
+FROM node:4.2.4-wheezy
+MAINTAINER Mark Stemm (mstemm@jut.io)
+
+RUN mkdir -p /opt/juttle-engine
+WORKDIR /opt/juttle-engine
+ADD package.json package.json
+RUN npm install
+COPY . .
```

Comment:
I'm no docker expert but it seems to me you should do this before running npm install, then you don't need the `ADD package.json package.json` line above, right?